Yoast/wordpress-seo
# make yoast components available on 'window.yoast.components'

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Adding the yoast components to the global scope of the browser.

## Test instructions

This PR can be tested by following these steps:

* install this branch of the plugin. (`yarn && composer install && grunt`)
* Verify in the browser console that `window.yoast` and `window.yoast.components` return an actual value like below:
<img width="667" alt="edit_post_ _qwerty_ _wordpress" src="https://user-images.githubusercontent.com/5509155/43828328-809ffa54-9afc-11e8-897d-5a71e6e73326.png">


## Quality assurance


Fixes #10569
